.golangci.yml

```diff
@@ -18,9 +18,13 @@
 # options for analysis running
 run:
   issues-exit-code: 1
-  skip-dirs-use-default: true
```

Comment:
This is independent of what we specify in the `skip-dirs`: it specifies that the check should not be run against dir like vendor etc. We should leave it, just to not depend on the defaults from the code.